policy.go

```diff
@@ -26,11 +26,14 @@ const (
 func calcMinRequiredTxRelayFee(serializedSize int64, minRelayTxFee btcutil.Amount) int64 {
 	// Calculate the minimum fee for a transaction to be allowed into the
 	// mempool and relayed by scaling the base fee (which is the minimum
-	// free transaction relay fee).  minRelayTxFee is in Satoshi/KB, so
-	// divide the transaction size by 1000 to convert to kilobytes.  Also,
-	// integer division is used so fees only increase on full kilobyte
-	// boundaries.
-	minFee := (1 + serializedSize/1000) * int64(minRelayTxFee)
+	// free transaction relay fee). minTxRelayFee is in Satoshi/KB so
+	// multiply by serializedSize (which is in KB) and divide by 1000 to get
+	// minimum Satoshis.
+	minFee := int64(minRelayTxFee)
+	// If we have more than 1KB calculate the new minFee
+	if serializedSize > 1000 {
```

Comment:
This does not match the behavior.  It should be:

```Go
minFee := (serializedSize * int64(minRelayTxFee)) / 1000
if minFee == 0 && minRelayTxFee > 0 {
	minFee = minRelayTxFee
}
```

For example, let's take some representative values.  A serialized transaction size of 284 bytes and a min fee of 5000.

The code in this PR as it is now would result in `minFee = 5000`.  However it should be `minFee = 1420`.

In fact, I think it would be  a good idea to add this specific test to `TestCalcMinRequiredTxRelayFee`.